android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/UnityAndroidPayFragment.java

```diff
@@ -197,4 +198,8 @@ public void onWalletRequestCancel(int requestCode) {
     public void setSessionCallbacks(AndroidPaySessionCallbacks callbacks) {
         sessionCallbacks = callbacks;
     }
+
+    public void setGoogleClient(GoogleApiClient client) {
```

Comment:
protected scope? `void setGoogleClient(GoogleApiClient client)` as it only used for testing